tests/Backend/Exasol/TableDefinitionOperationsTest.php

```diff
@@ -189,8 +190,70 @@ public function testDataPreviewForTableDefinitionWithDecimalType()
 
     public function testAddColumnOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed tables");
-        $this->_client->addTableColumn($this->tableId, 'newColumn');
+        $tableDefinition = [
+            'name' => 'my-new-table-add-column',
+            'primaryKeysNames' => ['id'],
+            'columns' => [
+                [
+                    'name' => 'id',
+                    'definition' => [
+                        'type' => 'INT',
+                    ],
+                ],
+                [
+                    'name' => 'column_decimal',
+                    'definition' => [
+                        'type' => 'DECIMAL',
+                        'length' => '4,3'
+                    ],
+                ],
+            ],
+        ];
+
+        $sourceTableId = $this->_client->createTableDefinition($this->getTestBucketId(self::STAGE_IN), $tableDefinition);
+
+        $firstAliasTableId = $this->_client->createAliasTable($this->getTestBucketId(self::STAGE_IN), $sourceTableId, 'table-1');
+        $secondAliasTableId = $this->_client->createAliasTable($this->getTestBucketId(self::STAGE_IN), $firstAliasTableId, 'table-2');
+
+        $this->_client->addTableColumn($sourceTableId, 'added_column', [
```

Comment:
osobne bych tam rad videl i vic datovych typu (zase po jednom z basetype)  a nejakej nevalidni vstup

Reply:
a mozna jeste scenar pridavani typovaneho sloupce na netypovanou tabulku v EXA/SNPS a pak na SNFLK/RS  . Proste at mame ty edgecasy pokryte

Reply:
typovany sloupec ti na typovanu tabulku, prejde uz teraz v mastru pretoze sa tam berie len ten parameter name a api nevaliduje ci posielas nieco naviac. Pridal som test teda ktory skusi pridat netypovany stlpec typovanej tabulke co by malo spadnut.

Reply:
- pridal som viac datovych typov, obecne by si to zasluzili teda aj testy ktora tam uz boli na pociatku
- nevalidny vstup testuju unit testy na req obj
- plus som pridal test ze nejde pridat typovanej tabulke stlpec ktory nema typy